src/librustc_typeck/error_codes.rs

```diff
@@ -1588,6 +1588,38 @@ fn bar(foo: Foo) -> u32 {
 ```
 "##,
 
+E0183: r##"
+Fn* traits (`unboxed closures`) are experimental. 
+See [tracking issue #29625][iss29625] for the status of the feature.
+
+Functions must have exactly one (non self) argument, a tuple representing
+the argument list [1]. 
+
+Here's an example of this error:
```

Comment:
Please replace with "Erroneous code example:".

Reply:
Forgot this one. *facepalms*